using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md

```diff
@@ -4,12 +4,23 @@
 Anomaly Detection is currently in beta. Please read the documentation carefully.
 {% endhint %}
 
-Anomaly Detection is a governance tool which detects when cloud services significantly deviate from their projected spend. This feature, as well as [Forecasting](anomaly-detection.md#forecasting), are powered by the Anomaly Detection Container, which is enabled in Kubecost by default. Kubecost samples spend data from the last seven days to determine of a service's spend has become anomalous.
+Anomaly Detection is a governance tool which detects when cloud services significantly deviate from their projected spend. This feature, as well as [Forecasting](anomaly-detection.md#forecasting), are powered by the Forecasting Container, which is enabled in Kubecost by default.
+
+{% hint style="info" %}
+Anomaly Detection requires at least one cloud provider integration in order to detect anomalies in service spend. See our [Cloud Integration Guide](/install-and-configure/install/cloud-integration/README.md) to get started.
+{% endhint %}
 
 ![Anomaly detection](/images/anomalydetection.png)
 
+The Forecasting Container can be disabled by setting the Helm flag:
+
+```
+forecasting:
+    enabled: true

```

Comment:
should just be two spaces and false:
```
  enabled: false
```

Reply:
What do you mean by two spaces?

Reply:
I think he means the indentation of `enabled: false`... incorrect YAML indentation spacing can cause issues.

Reply:
Added an update comment with suggested changes.